doc/source/dev/contributor/building_faq.rst

```diff
@@ -80,10 +80,25 @@ Now you have a branch in your local development area to code review in Python.
 How do I deal with Fortran ABI mismatch?
 ========================================
 
+_For ``meson`` based builds (new in 1.9.0):_
+
+Use the ``-Duse-g77-abi=true`` build option. E.g.,::
+
+    $ meson setup build -Duse-g77-abi=true
+
+A more complete example, also configuring the BLAS/LAPACK libraries and picking
+a better Python install behavior (this is what conda-forge could be using for
+example)::
+
+    $ meson setup builddir -Duse-g77-abi=true -Dblas=blas -Dlapack=lapack -Dpython.install_env=auto
+    $ meson install -C builddir
+
+_For ``distutils`` based builds:_
+
 Some linear algebra libraries are built with G77 ABI and others with
 GFortran ABI, and these two ABIs are incompatible. Therefore, if you
-build scipy with `gfortran` and link to a linear algebra library, like
-MKL, which is built with G77 ABI, then there'll be an exception or a
+build SciPy with ``gfortran`` and link to a linear algebra library, like
+MKL, which is built with g77 ABI, then there'll be an exception or a
 segfault. SciPy fixes this by using the CBLAS API for the few
 functions in the BLAS API that suffers from this issue.
```

Comment:
Since it's for both `distutils` and `meson` builds, move this to the top of the subsection?

Reply:
Good point, done.